requirements-dev.txt

```diff
@@ -40,3 +40,4 @@ requests
 einops
 transformers
 mlflow
+matplotlib
```

Comment:
Could you please help also add `matplotlib` to other config files for optional import packages?
Like `pip install monai[matplotlib]`.

Thanks.